jekyll/_data/sidenav.yml

```diff
@@ -116,10 +116,10 @@ en:
     - name: Machine
       children:
         - name: Overview
-          link: 2.0/executor-types/
+          link: 2.0/executor-types/#using-machine
           hash: using-machine
         - name: Pre-installed software
-          link: 2.0/docker-to-machine
+          link: 2.0/docker-to-machine#pre-installed-software
```

Comment:
I don't think we should change the URLs in that file. Or at least not as part of that PR. Any reason you tweaked that?

Reply:
the yml file link did not match the link in the <a> tags so it seems like an inconsistency issue. The 'should go through sidebar yml to see if all links exist in EN then JA' test just goes through and makes sure all links exist. I had a hard time trying to get cypress to look for a href that only contains the link even if its a partial link. I could only get it to work if it was exact match. I can look into it further but that is the reason for the change

Reply:
Hmm, I think it's okay to change this, but if we are changing it we are effectively doubling up on hashes with the use of the hash key on line `120`. This was originally used to construct urls to get around our issues with using query-params + local-storage to indicate where the user is in the docs when they reload the page and the sidebar jumps to the document they are on. It seems that we no longer use the query params (I think @rsylvian might have refactored that a while ago). If that's the case, we could make this change, but we should remove the use of `hash` and then get rid of it where it appears in sidebar.html:

```
                  <a class="{% if url_slug == link_slug %} active {% endif %}"
                    href="{{ site.baseurl }}/{{ item.link }}{% if item.hash %}#{{item.hash}}{% endif %}" data-section="{{ section_slug }}">
                    {{display_name}}
                  </a>
```

Reply:
Good catch! I did not see double # in the html or the test would fail but I will look into this code if can now be removed

Reply:
I think since we want to do a source of truth it be good to not make these changes after the fact from sidenav.yml

Reply:
I didn't realize sidenav had `hash: using-docker` so I don't have to alter sidenav.yml itself my mistake